### PR TITLE
Application components created

### PR DESCRIPTION
**Pull request by leovaldez08**
Created necessary HomePage components like NavBar, HeroSection, Carousel and Footer.

### CODE REVIEW

File: FrontEnd/index.html [hunk 1]
```diff
@@ -2,9 +2,15 @@
...
+    <!-- <link rel="icon" type="image/svg+xml" href="/vite.svg" /> -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React</title>
+    <title>PromptVers</title>
```
Comment (yash0212) [inline FrontEnd/index.html]: Is there a typo here?

**Review by yash0212 — COMMENTED** [review]
Please respond to the comment and add a fix if required

File: FrontEnd/index.html (hunk 1 above)
Comment (leovaldez08) [reply FrontEnd/index.html]: Fixed now!

**Review by jithin-kalvian — APPROVED** [review]
LGTM

**Pull request merged**